src/decode.ts

```
@@ -383,8 +383,11 @@ export class EntityDecoder {
                     (current & BinTrieFlags.FLAG13) === 0
                 ) {
                     this.result = this.treeIndex;
-                    this.consumed += this.excess;
-                    this.excess = 0;
+                    // `excess` started at 1 to count the leading `&`, which is
+                    // already in `consumed`; subtract it so the run's
+                    // characters are not counted twice.
```

ESLint

[error] Expected a block comment instead of consecutive line comments.

(multiline-comment-style)